tf_encrypted/keras/engine/base_layer.py

```diff
@@ -1,10 +1,7 @@
 """Includes base classes used by all layer types."""
 
-from typing import List, Optional
 from abc import ABC, abstractmethod
-
-from tf_encrypted.protocol.pond import TFEVariable
-from tf_encrypted.protocol.protocol import get_protocol, Protocol
+from tf_encrypted import get_protocol
```

Comment:
consider adding blank link here to separate standard packages and project packages